articles/aks/faq.md

```diff
@@ -189,7 +189,7 @@ While AKS has resilience mechanisms to withstand such a config and recover from
 
 ## Can I use custom VM extensions?
 
-No, AKS is a managed service, and manipulation of the IaaS resources is not supported. To install custom components, etc. please leverage the Kubernetes APIs and mechanisms. For example, leverage DaemonSets to install required components.
+Other than the Log analytics agent (which is supported because it is a Microsoft managed extension) No, AKS is a managed service, and manipulation of the IaaS resources is not supported. To install custom components, etc. please leverage the Kubernetes APIs and mechanisms. For example, leverage DaemonSets to install required components.
```

Comment:
Change to: 
> The Log analytics agent is supported since it is an extension managed by Microsoft. Otherwise no, AKS is...

Reply:
@zr-msft I don't see this one in the published FAQ?